CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # Changelog
 
+## [1.9.0] - 2024-02-22
+
+### Added
+
+- `uniqueid` CredentialComposer plugin that adds the x509UniqueIdentifier attribute to workload X509-SVIDs (#4862)
+- Agent's Admin API has now a default location defined (#4856)
+- Partial selectors from workload attestation are now logged even if the client hangs up a connection to the agent (#4846)
+- X509-SVIDs minted by SPIRE can now include wildcards in the DNS names (#4814)
+
+### Changed
+
+- CA journal data is now stored in the datastore, removing the on-disk dependency of the server (#4690)
+- `aws_kms`, `azure_key_vault`, and `gcp_kms` KeyManager plugins no longer require storing metadata files on disk (#4700)
+- Bundle endpoint refresh hint now defaults to 5 minutes (#4847, #4888)
+- Wait for all RPC handlers to drain when unloading built-in plugins (#4820)
+- Entry cache hydration is now done with paginated requests to the datastore when listing entries (#4721)
+- Node attestors that can be re-attested now go through re-attestation by default (#4791)
+- SPIRE Agent now uses the LRU cache by default (#4773)
+- Small documentation improvements (#4764, #4787)
+- Read-replicas are no longer used for initial events-based cache load (#4868)
+- Events based entry cache is now paginated on startup (#4826)
```

Comment:
I think line 18 can serve to cover this change for both caches. Maybe we just add this PR number to that line?

Reply:
Yes, that makes sense.